data/database/slack.json

```diff
@@ -20,6 +20,18 @@
         "unread": {
             "original": "slack-taskbar-unread.png",
             "theme": "slack-indicator-unread"
```

Comment:
You can use symlinks keyword here to create links to other icons instead of duplicating them. You can take a look at other files implementing that

Reply:
My mistake, good to know. Fixed.